src/Symfony/Component/Form/FormError.php

```diff
@@ -187,4 +187,18 @@ public function unserialize($serialized)
     {
         list($this->message, $this->messageTemplate, $this->messageParameters, $this->messagePluralization, $this->cause) = unserialize($serialized);
     }
+
+    /**
+     * @see JsonSerializable::jsonSerialize()
+     */
+    public function jsonSerialize()
+    {
+        return array(
+            'message' => $this->message,
+            'messageTemplate' => $this->messageTemplate,
+            'messageParameters' => $this->messageParameters,
+            'messagePluralization' => $this->messagePluralization,
+            'cause' => $this->cause,
```

Comment:
@operandom What do you mean? `$this->cause` can be an instance of `ConstraintViolationInterface` or even an `Exception`, how would that work here?